example/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -52,5 +55,37 @@ public void onCreate() {
     // If you opted-in for the New Architecture, we enable the TurboModule system
     ReactFeatureFlags.useTurboModules = BuildConfig.IS_NEW_ARCHITECTURE_ENABLED;
     SoLoader.init(this, /* native exopackage */ false);
+    initializeFlipper(this, getReactNativeHost().getReactInstanceManager());
```

Comment:
Let's not bring the Flipper back - I removed it from example app as it was unused and I was trying to reduce the codebase.
Let's also remove the whole `initializeFlipper` method definition and `ReactNativeFlipper.java` file.